mod.yaml

```diff
@@ -19,6 +19,7 @@ Packages:
 	$yupgi_alert: yupgi_alert
 	$ra: ra
 	$cnc: cnc
+	$as: as
```

Comment:
This should also be ./mods/as at best.

$x: x notation requires a mod with it's mod.yaml which isn't released at AS, only the source code of the plugin.